.github/workflows/load-test.yml

```
@@ -0,0 +1,104 @@
+
+      - name: Run CI load test
+        id: load-test
+        working-directory: backend
+        run: |
+          node scripts/ci-load-test.js 2>&1 | tee load-test-output.txt
+
+      - name: Check thresholds from JSON output
+        id: check-thresholds
+        run: |
+          python3 << 'EOF'
+import json, os
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)